ex2_hal/iris/equipment_handler/include/iris_spi.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+/**
+ * @file    spi.h
+ * @author  Jenish Patel
+ * @date    2022-05-03
+ */
+
+#ifndef INCLUDE_IRIS_SPI_H_
+#define INCLUDE_IRIS_SPI_H_
+#endif /* INCLUDE_IRIS_SPI_H_ */
+
+#include "FreeRTOS.h"
+
+typedef enum {
+    IRIS_ACK = 0,
+    IRIS_NACK = 1,
+    IRIS_LL_ERROR = 2,
+} IrisLowLevelReturn;
+
+#define DUMMY_BYTE 0xFF
+
+void iris_spi_init();
+static void spi_send(uint16_t *tx_data, uint16_t data_length);
```

Comment:
It doesn't look like you use these functions outside of the associated C file here. You should not have them declared in the header in that case. 

If you want to leave them here, they should not be static. Also, they should have a unique name like iris_spi_send

Reply:
Should be fixed now